editor/src/messages/input_mapper/utility_types/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod input_keyboard;
 pub mod input_mouse;
+pub mod input_time;
```

Comment:
I'd suggest adding that struct to `misc.rs` instead of adding this new file for only 20 lines.